src/main/scala/com/databricks/sparkdl/DeepImageFeaturizer.scala

```diff
@@ -95,14 +104,20 @@ class DeepImageFeaturizer(override val uid: String) extends Transformer with Def
     this
   }
 
+  def setResizeFlag(value: String): this.type = {
+    set(scaleHint, value)
```

Comment:
I think we always use the naming convention `setX` for param `X`. Could this be renamed to `setScaleHint` or could we rename the param as `resizeFlag`.

Reply:
Yes, I've renamed resizeFlag to scaleHint and forgot to change the method. Good catch